docs_nnx/guides/transforms.md

```diff
@@ -51,14 +51,14 @@ weights = Weights(
   kernel=random.uniform(random.key(0), (10, 2, 3)),
   bias=jnp.zeros((10, 3)),
 )
-x = jax.random.normal(random.key(1), (10, 2))
+count = jax.random.normal(random.key(1), (10, 2))
```

Comment:
Why renamed it to `count`? `x` makes more sense as a model input.

Reply:
Thanks for catching this! There's a vscode bug when renaming variables in jupyter notebooks. Reverted back to `x`.